g11n-ws/vip-manager-i18n/src/main/java/com/vmware/vip/core/conf/SwaggerConfig.java

```diff
@@ -89,7 +89,7 @@
 	@ConditionalOnProperty(value = "vipservice.authority.enable")
```

Comment:
Should change this to authentication?

Reply:
this just change it in swaggerUI.  this use in configuration